src/main/java/TaskManager.java

```diff
@@ -66,27 +67,29 @@ public void addTask(String option, String taskDescription){
         description = description.trim();
         String time = taskDescription.substring(sepIndex + 3);
         time = time.trim();
-        if(option.equals("deadline")){
+        if(option.equals("deadline")) {
             tasks[taskCount++] = new Deadline(description, time);
-        }
-        else if(option.equals("event")){
+        } else if(option.equals("event")) {
```

Comment:
Great job on consistently maintaining the right format on your if-else statements